pkg/devfile/parser/parse.go

```diff
@@ -397,11 +400,24 @@ func parseFromURI(importReference v1.ImportReference, curDevfileCtx devfileCtx.D
 func parseFromRegistry(importReference v1.ImportReference, resolveCtx *resolutionContextTree, tool resolverTools) (d DevfileObj, err error) {
 	id := importReference.Id
 	registryURL := importReference.RegistryUrl
+	destDir := "."
+
+	// contains the downloaded stack from the registry
+	tempDir, _ := ioutil.TempDir("", "")
+	defer os.RemoveAll(tempDir)
+
 	if registryURL != "" {
-		devfileContent, err := getDevfileFromRegistry(id, registryURL, importReference.Version)
+		err = getStackFromRegistry(id, registryURL, tempDir, destDir)
 		if err != nil {
 			return DevfileObj{}, err
 		}
+		devfileContent, err := getDevfileFromDir(tempDir)
```

Comment:
should be `destDir`?
```suggestion
		devfileContent, err := getDevfileFromDir(destDir)
```